.coafile

```diff
@@ -0,0 +1,41 @@
+[Default]
+ignore = .tox/**, .gitignore, .gitreview, .gitmodules, **/provisioning/roles/**
```

Comment:
There aren't any .sh files in this repo (`,find . -name "*.sh" `) and thinking about Puppet I can't imagine we'll end up writing shell helpers or something, so I guess it can be removed.

> Sorry, shouldn't have blindly copied the coafile. 

np, just include "WIP" (work in progress) in the commit message if you're planning on updating the commit